app/models/package_manager/pypi/json_api_project.rb

```diff
@@ -130,7 +130,13 @@ def deprecation_status
         is_deprecated = false
         message = nil
 
-        if classifiers.include?(CLASSIFIER_INACTIVE)
+        stable_releases = releases.reject(&:prerelease?)
+        latest_stable = stable_releases.last
+
+        if stable_releases.all?(&:yanked?)
```

Comment:
Could you add a code comment to link this to https://github.com/librariesio/libraries.io/blob/aed17eeb7130ead25f82b8508b9c78f7f9706d6c/app/models/package_manager/pypi/version_processor.rb#L16-L30, as this is the other place (that I'm aware of) that is processing pypi yanked releases and we probably don't want the two places doing this to have any conflicts